.github/workflows/job_keras3_backend.yml

```
@@ -90,7 +90,8 @@ jobs:
       - name: Install test dependencies
         working-directory: ${{ env.KERAS_REPO }}
         run: |
-          pip install -r requirements.txt --upgrade
+          pip install tf_keras==2.19.0
+          pip install -r requirements.txt
```

```suggestion
          pip install -r requirements-common.txt
```
should be better and avoid installing TF, PT, etc.